README.md

```diff
@@ -47,7 +47,8 @@ The action supports the following inputs:
   [binary packages](https://github.com/zeek/zeek/wiki/Binary-Packages):
   `zeek` for the latest release, `zeek-lts` for the latest
   long-term-support release, and `zeek-nightly` for the latest nightly
-  Zeek build.
+  Zeek build. Starting with zeek-6.0.0 we also support explicitly versioned
+  packages like `zeek-6.0`.
```

Comment:
Should also work with zeek-5.0:
```
root@871c314f76b5:/# dpkg -l | grep zeek-5.0
ii  zeek-5.0                    5.0.10-0                       amd64        Zeek is a powerful framework for network analysis and security monitoring.
```

Related to the other comment, maybe tighten this to something like "LTS release versions can be pinned using `zeek-6.0`".

Reply:
Ah I see, works with `zeek-5.0`, but not with e.g., `zeek-5.1`. That's too bad.

I'll add another CI job for 5.0.